runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -3916,6 +3917,131 @@ J9::CodeGenerator::collectSymRefs(
    return true;
    }
 
+  /** \brief
```

Comment:
this doesn't seem brief...

Reply:
Fixed in https://github.com/eclipse/openj9/pull/4721/commits/657326068c58cbe715a3d1d1af251eaa1ab3d371